### ISSUE TITLE
tax displaying

### ISSUE THREAD

**Issue by maikel-exitable** [mglaman/commerce_demo#15]
Hi, i am trying to use your module to enable a dutch vat rule. First i activated the module, and then added a new rule for the dutch vat.

```
// Apply VAT for Germany.
	    if ($address->getCountryCode() == 'NL') {
		    $vat_adjustment = new Adjustment([
			    'type' => 'tax',
			    'label' => t('VAT'),
			    'amount' => $order->getTotalPrice()->multiply('0.21'),
		    ]);
		    $order->addAdjustment($vat_adjustment);
	    }
```

it gets triggered, but i get the notice that "tax is an invalid adjustment type". Am i missing something? Did i destroy anything with the addition of above code?

Probably just overlooking something, but unable to see it.

Thanks in advance!
Maikel

**Comment by mglaman** [comment]
No, the module is just a bit out of date! We added strict adjustment types
defined in a YAML. I'm hoping to spend some time on flight to, and at,
Drupal camp London to update the demo!

**Comment by maikel-exitable** [comment]
Thanks for the quick reply. 

If there is a direction you can point me in (something to read up on concerning the outdated part), i am happy to help and add / adjust the code.

**Comment by mglaman** [comment]
If you look at mglaman/commerce_avatax there's an adjustment_type.yml,
basically we need a copy of that and change the "type" in processor to
"sales_tax", or we'll make a VAT one too.

Thanks for the offer to help!

**Comment by maikel-exitable** [comment]
I tried what you said and when just using vat (from the example) and this as the file

commerce_demo.commerce_adjustment_types.yml
```
tax:
  label: VAT
  has_ui: true
  weight: 100
```

it works like a charm. Thanks again!

**Comment by mglaman** [comment]
Great. Would you mind opening a PR for me? You might even be able to pull
it off solely from the GitHub UI editor. If not, I'll circle am back to
this around my afternoon :)

**Comment by maikel-exitable** [comment]
Sure!
